### PR TITLE
Improve man page and --help

### PR DESCRIPTION
**Pull request by mtrmac**
Should, at least partially, address #134 and #137.

### CODE REVIEW

**Comment by rhatdan** [comment]
These LGTM.

But I would prefer if we rewrote these all in markdown, so they could easily be displayed on web sites and it would make it easier for people to contribute.

More examples is always helpful also.

File: man1/skopeo.1 [hunk 1]
```diff
@@ -6,121 +6,158 @@
...
 .TP
-.B standalone-sign
+\fBskopeo standalone-sign\fP \fImanifest\fP \fIdocker-reference\fP \fIkey-fingerprint\fP \%\fB--output\fP|\fB-o\fP \fIsignature\fP
 Create a signature using local files.
-This is primarily a debugging tool and should not be part of your normal operational workflow.
-.sp
-.B manifest
-path to file containing manifest of image
+This is primarily a debugging tool, or useful for special cases,
+and usually should not be a part of your normal operational workflow;
```
Comment (rhatdan) [inline man1/skopeo.1]: I thought we were going to remove this message.  Lets not say it is just a debugging tool.

Comment (mtrmac) [inline man1/skopeo.1]: That’s why there is the new “or useful for special cases” case.

**Comment by mtrmac** [comment]
> But I would prefer if we rewrote these all in markdown, so they could easily be displayed on web sites and it would make it easier for people to contribute.

Sure, that could be useful; it is not on the critical path for signing, though, so I have filed #151 to track the markdown conversion.

Thanks for the review!

**Pull request merged**